tests/integration/modules/file.py

```diff
@@ -293,6 +293,16 @@ def test_source_list_for_list_returns_local_file_proto_from_dict(self):
         else:
             self.assertItemsEqual(ret, ['file://' + self.myfile, 'filehash'])
 
+    def test_file_line_changes_format(self):
+        '''
+        Test file.line changes output formatting.
+
+        Issue #41474
+        '''
+        ret = self.minion_run('file.line', self.myfile, 'Goodbye',
+                              mode='insert', after='Hello')
+        self.assertIn('Hello\n+Goodbye', ret)
```

Comment:
As mentioned in previous comments, we should be careful with Windows with this change. I haven't tested the code change above, but as this test stands, the test itself will fail on Windows. Something like `os.sep` should be used here. See #43943 for an example.

Reply:
Should be:
```
self.assertIn('Hello' + os.linesep + 'Goodbye', ret)
```
to work on Windows.